pyproject.toml

```diff
@@ -11,7 +7,25 @@ line-length = 119
 # Ignore import violations in all `__init__.py` files.
 [tool.ruff.per-file-ignores]
 "__init__.py" = ["E402", "F401", "F403", "F811"]
+"manim_animations/*" = ["ALL"]
 
 [tool.ruff.isort]
 lines-after-imports = 2
 known-first-party = ["accelerate"]
+
+[tool.ruff.format]
+# Like Black, use double quotes for strings.
```

Comment:
Let's just have this all under "Configurations similar to black"

Reply:
In that case, I'll just preferably remove the explicit configuration that's repeating the defaults; Ruff is by default Black-compatible. I borrowed this verbiage from https://github.com/huggingface/transformers/pull/27144.